dev-docs/show-native-ads.md

```diff
@@ -172,7 +174,7 @@ A native `image` ad unit can be set up in the manner below:
 
       const adUnits = [{
         code: 'adUnit-code',
-        mediaTypes: { native: { type: 'image' } }
+        mediaTypes: { native: { image: { sizes: [[100, 50]] } } }
```

Comment:
`type: 'image'` expands to a set of pre-defined assets and doesn't take subparams, this line should be reverted